packages/kotti-ui/source/kotti-form-controller-list/tests.ts

```diff
@@ -57,7 +57,7 @@ const getField = (
 	// eslint-disable-next-line @typescript-eslint/no-explicit-any
 	wrapper: Wrapper<any>,
 	index: number,
-): KottiField.Hook.Returns<string | null, string | null> =>
+): KottiField.Hook.Returns<string | null> =>
```

Comment:
supports value `string | null` even tho it's never on KottiField

extend the props passed in tests